packages/input-datepicker/test/lion-input-datepicker.test.js

```diff
@@ -98,14 +98,19 @@ describe('<lion-input-datepicker>', () => {
       expect(elObj.overlayController.isShown).to.equal(false);
     });
 
-    it('closes the calendar via outside click', async () => {
+    it('closes the calendar via outside mousedown event', async () => {
       const el = await fixture(html`<lion-input-datepicker></lion-input-datepicker>`);
       const elObj = new DatepickerInputObject(el);
       await elObj.openCalendar();
       expect(elObj.overlayController.isShown).to.equal(true);
 
       document.body.click();
       await aTimeout(0);
```

Comment:
```suggestion
```
Nitpick:

Can these two lines be removed if we're testing with a mouse down event instead of `.click()`?